src/yml/social.yml

```
@@ -0,0 +1,5 @@
+github_username: idvogados
+facebook_username: idvogados
+twitter_username: idvogados
```

Atualmente, nosso user do twitter é o @idvogados_ofc
```suggestion
twitter_username: idvogados_ofc
```